### ISSUE TITLE
Changing any setting in "General settings" asks for a password

### ISSUE THREAD

**Issue by mattab** [piwik/piwik#20504]
in System > General Settings,  whenever any "Save" button is clicked, currently it asks for a password confirmation, which is not user friendly and not expected.

I assume it's a bug and not sure if it's maybe a recent regression?

Reproduce:

- Login as super user.
- Go to general settings
- Try Changing the newly introduced settings for MultiChannel plugin
- It does ask for Password check
- Expected: it does not ask for password check.

Reproduced in: web.innocraft.cloud/index.php?idSite=12&module=CoreAdminHome&action=generalSettings#/MultiChannelConversionAttributionPluginSettings

**Comment by heurteph-ei** [comment]
Isn't if for security purposes?
When I do some money transfer, my banking app ask me for password confirmation...

**Comment by tsteur** [comment]
This was added on purpose for security reasons. General settings are changed rarely, and can have wide impacts. So it's better to request a password for these changes for better security.

**Comment by sgiehl** [comment]
Personally I think it's correct to ask for a password. But it's indeed a bit annoying, that every plugin setting has it's own section that needs to be saved separately. What in my opinion could improve that, would be to remove/hide all save buttons and have one save button in the end (or maybe in a fixed position in the lower screen, so you don't need to scroll down). Clicking that button would then show one password prompt and trigger all changed settings to be saved.

**Comment by heurteph-ei** [comment]
I totally agree with @sgiehl . Once, I wanted to save some settings thanks to a wrong button... Then it did not save anything!
+1 for the fixed position!

**Comment by Stan-vw** [comment]
@Javi-Ormaechea any ideas from your end on whether the current state (a save button at the bottom of every config page, which have varying lengths) makes sense or whether we can improve the UX here?

**Comment by sgiehl** [comment]
My suggestion would be something like a fixed bar at the bottom. e.g. like
![image](https://user-images.githubusercontent.com/1579355/227904394-4c6c3f64-a485-4def-b43e-7750761918ee.png)
(maybe with a better looking color 😄 )

**Comment by Javi-Ormaechea** [comment]
It would be interesting to know how many users are finding this a problem. Definitely a solution could look like this (similar to @sgiehl proposal) 

![Screenshot 2023-03-28 at 11 44 43 AM](https://user-images.githubusercontent.com/109315616/228083468-76d2a3ae-eca5-4bf1-b614-957e7537b3f6.png)

or like below, with the buttons placed after all the sections.

![Screenshot 2023-03-28 at 12 31 48 PM](https://user-images.githubusercontent.com/109315616/228089833-f1b7adce-065a-4b6f-ab4b-297dae4e22a3.png)

**Comment by rr-it** [comment]
Just implement a time window of 5 minutes: Only ask for password once every 5 minutes.

TYPO3 has actually implemented the "sudomode/system maintainer" password check like this.

And AFAIK the "National Cyber Security Center (NCSC) of Switzerland" who reported a security issue looks to be just fine with this 5-minutes-window.

Edit: For a user password-change there is no window and TYPO3 always asks for sudo-mode password confirmation.

See:
* https://typo3.org/security/advisory/typo3-core-sa-2025-013
* https://github.com/typo3/typo3/commit/1d184b2c89c
* https://github.com/TYPO3/typo3/blob/7ae330441bfca1d63cc4bfa14a9b9d9f0890f6e3/typo3/sysext/backend/Classes/Security/SudoMode/Access/AccessLifetime.php#L39
